### DISCUSSION TITLE
Headless+SAAS example/recipy?

### DISCUSSION THREAD

**Discussion by marcusjm** [OrchardCMS/OrchardCore#11010] [Q&A] [answered]
Is there a recipy that combines SAAS and Headless? Say I'd like to create many static sites run from one setup.

**Comment by marcusjm** [comment, marked as answer]
Found out myself.

**Reply by Piedone** [reply]
Would you share your solution so others can learn from it?

**Reply by hishamco** [reply]
https://github.com/OrchardCMS/OrchardCore/blob/e13cda97e036205ddb7c2fa1c655cc07385bbc2d/src/OrchardCore.Themes/TheAdmin/Recipes/headless.recipe.json

**Reply by marcusjm** [reply]
Thank You.
